junebug/channel.py

```diff
@@ -259,6 +259,7 @@ def _application_config(self):
             'redis_manager': self.config.redis,
             'inbound_ttl': self.config.inbound_message_ttl,
             'outbound_ttl': self.config.outbound_message_ttl,
+            'message_rate_bucket': self.config.inbound_message_rate_bucket,
```

Comment:
I would vote for just `message_rate_window` or `metric_window` everywhere, unless there is a reason to have separate window sizes?